mappings/UofM.py

```diff
@@ -16,6 +16,7 @@ def createMapping(self):
                 ('ISBN', '{0}|isbn'),
                 ('OCLC', '{0}|oclc')
             ],
+            'rights': ('DRB Rights Classification', '{0}||||'),
```

Comment:
Is the rights classification specific to DRB or is it general? Could this just be Rights Classification?

Reply:
This name is the name of the column in AIrtable with the rights classification. When converting the csv format of Airtbale into a json file, the name remains as a key which is used in this mapping to get the value of that key.